linuxrc_hostname.md

```diff
@@ -1,31 +1,76 @@
 # hostname setting in installation/rescue system
 
-There are 3 cases:
+The hostname can be set (in order of precedence):
 
-1. no hostname is set
+1. via `hostname` boot option
+2. via DHCP (either from HOSTNAME or FQDN)
+3. Default hostname set by Yast
```

Comment:
We should take into account that this is a linuxrc documentation, and thus, the previous documentation was still valid.

From my POV, main changes should address the write of /etc/hostname by YaST during a common installation as well as the proposal of the hostname by YaST at the end of the installation previously described.

What I mean is that linuxrc still sets the transient hostname ('install' or the one given by hostname='hostname').

Reply:
## Contradiction?

I see a contradiction between the first to paragraphs.

> ...this is a linuxrc documentation...

vs 
> ...main changes should address the write of /etc/hostname by YaST...

and
> ...the proposal of the hostname by YaST at the end of the installation...


## Suggestion

I think that it would be better to either

1. split the documentation between everything linuxrc related in openSUSE/linuxrc/linuxrc_hostname.md and everything Yast related in Yast/yast2-network? Yast/yast2-installer? and crosslink them,
2. or put everything in openSUSE/linuxrc/linuxrc_hostname.md. If not the doc will be technically true but confusing/misleading and useless.

> ...linuxrc still sets the transient hostname...
ah! I thanks for clarifying that :)

> ...[linuxrc] also enables the DHCLIENT_SET_HOSTNAME in case of configured dhcp interface...

Right, I forgot about that.

Reply:
> In general, I would split the documentation, leaving here only linuxrc responsibilities but not mixing with YaST workflows.

The document title is 'hostname setting in installation/rescue system' and I would like to stick to that here. As you see it's complex enough. :-)